tests/personio/test_personio_source.py

```diff
@@ -0,0 +1,35 @@
+import dlt
+import pytest
+
+from sources.personio import personio_source
+from tests.utils import ALL_DESTINATIONS, assert_load_info, load_table_counts
+
+
+@pytest.mark.parametrize("destination_name", ALL_DESTINATIONS)
+def test_all_resources(destination_name: str) -> None:
+    pipeline = dlt.pipeline(
+        pipeline_name="personio",
+        destination=destination_name,
+        dataset_name="shopify_data",
+        full_refresh=True,
+    )
+    # Set per page limit to ensure we use pagination
+    tables = ["employees", "absences"]
```

Comment:
are you not testing the table `attendances` because it is empty in the test account? maybe add a couple of samples and test it too? i can help with it if you have no time